docs: asset folder clearing hint (#18320)
raycast/extensions: docs/information/file-structure.md

The optional `assets` folder can contain icons that will be packaged into the extension archive. All bundled assets can be referenced at runtime. Additionally, icons can be used in the `package.json` as extension or command icons.

{% hint style="info" %}

> During development the `assets` folder of the extension in raycast will be cleared when starting the extension via `npm run dev` to ensure you get the same behavior the user would get when installing the extension.

Reviewer: Let's clarify which `assets` folder would be cleared. Because there are two `assets` folders. One is our project source `assets` folder, and the another one is `~/.config/raycast/extensions/YOUR_EXTENSION/assets`.

I think we can use `environment.assetsPath` instead in the documentation.

Author: Great suggestion, what about this phrasing then?
```suggestion
During development the `assets` folder for `environment.assetsPath` will be cleared when starting the extension via `npm run dev` to ensure you get the same behavior the user would get when installing the extension.
```
